database/create_user.sql

```diff
@@ -1,10 +1,18 @@
 CREATE TABLE users
 (
-    ID SERIAL PRIMARY KEY,
-    email VARCHAR(30),
-    password VARCHAR(30),
-    created_at TIMESTAMP,
-    modified_at TIMESTAMP
+    user_id SERIAL PRIMARY KEY NOT NULL,
```

Comment:
Not null not necessary, primary key enforces unique not null automatically.

Reply:
fix